docs/inso-cli/install.md

```diff
@@ -5,7 +5,18 @@ category: "Inso CLI"
 category-url: inso-cli
 ---
 
-Before you start, you will first need to install [Node.js](https://nodejs.org/en/download/) (recommended version 12.x LTS).
+{:.alert .alert-primary}
+**Note**: Inso CLI is currently only compatible with **Node.js 12**. If you need to work with multiple Node.js versions on your local machine, we recommend using [nvm](https://github.com/nvm-sh/nvm) to manage your versions.
```

Comment:
nvm is only for macOS and Linux :( Here's one for Windows: https://github.com/coreybutler/nvm-windows

Also, I think instead of a recommendation it's more of a "here are some options, buyer beware".

If we recommend something does that make us responsible for making sure it always works or if there are security issues in the future?

Lastly, it'll be good to say "12.x LTS" like before. That indicates any version of Node 12 that has Long Term Support.